### PR TITLE
Rewriting MTU configuration documentation to include __CNI_MTU__

### PR DESCRIPTION
**Pull request by insequent**
## Description

This is a relative minor update to the MTU Configuration page to represent current Kubernetes self-hosted manifests techniques.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
Updated MTU Configuration page to include references to __CNI_MTU__
```

### CODE REVIEW

**Comment by insequent** [comment]
Being the new guy, any input is much appreciated!

File: master/usage/configuration/mtu.md [hunk 1]
```diff
@@ -66,42 +66,47 @@ This should either be set on the standalone `libnetwork` service, or on the
...
 
-### Setting tunnel MTU through Felix Environment variable
+### Setting tunnel MTU with a Felix environment variable
 
 Passing in the environment variable `FELIX_IPINIPMTU` when running the
 `{{site.nodecontainer}}` container will set the MTU for Felix to use.
 
-When using the Kubernetes self-hosted manifests, the Felix derives this value from the `veth_mtu`
-field of the calico-config ConfigMap and is set to `1440` by default.
+When using the Kubernetes self-hosted manifests, the Felix derives this value from
```
Comment (emanic) [inline master/usage/configuration/mtu.md]: I realize that this error was present already, but while you're here, could you remove the incorrect article in front of Felix. It's just Felix, not the Felix. Thanks!

File: master/usage/configuration/mtu.md [hunk 2]
```diff
@@ -51,8 +51,8 @@ The user will also want to configure {{site.prodname}}'s IP-in-IP interface MTU
 IP-in-IP is enabled on the cluster. Refer to the MTU table at the top of the page
 to choose the value that matches your environment.
 
-> **Note**: The MTU on existing workloads will not be updated with these
-changes.  To have all workloads use the new MTU, they must be restarted.
+> **Note**: The MTU on existing workloads will not be updated with these changes. To update
+workload MTUs, see the section below corresponding to your plugin type.
```
Comment (emanic) [inline master/usage/configuration/mtu.md]: nit: s/below corresponding/that corresponds

**Comment by insequent** [comment]
Thanks for reviewing, @emanic. Let me know if see anything else.

File: master/usage/configuration/mtu.md [hunk 3]
```diff
@@ -66,42 +66,47 @@ This should either be set on the standalone `libnetwork` service, or on the
...
-### Setting tunnel MTU through Felix Environment variable
+### Setting tunnel MTU with a Felix environment variable
 
 Passing in the environment variable `FELIX_IPINIPMTU` when running the
 `{{site.nodecontainer}}` container will set the MTU for Felix to use.
 
-When using the Kubernetes self-hosted manifests, the Felix derives this value from the `veth_mtu`
-field of the calico-config ConfigMap and is set to `1440` by default.
+When using the Kubernetes self-hosted manifests, Felix derives this value from
+the `veth_mtu` field of the calico-config ConfigMap and is set to `1440` by default.
```
Comment (emanic) [inline master/usage/configuration/mtu.md]: nit: sentence is not quite correct, I know it's not your error but if you could:
s/and/which
that will make it much better.
Thanks @insequent !

Comment (insequent) [reply master/usage/configuration/mtu.md]: An easy fix, no problem whatsoever

**Comment by insequent** [comment]
Oops, didn't realize I misspelled `rewriting` in the commit. I'll forgo re-pushing unless you all want me to.

**Pull request merged**